src/Microsoft.VisualStudio.Services.Agent/TraceManager.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Concurrent;
 using System.Diagnostics;
 using Agent.Sdk.SecretMasking;
+using Microsoft.TeamFoundation.DistributedTask.Logging;
```

Comment:
>Logging [](http://example.com/codeflow?start=47&length=7)

Why did you have to add this using directive?

Reply:
Yes, and yes. :)

Reply:
Fixed.